pumpkin-lib/src/propagators/cumulative/time_table/time_table_per_point.rs

```diff
@@ -252,6 +253,7 @@ mod tests {
                 CumulativeOptions {
                     allow_holes_in_domain: false,
                     explanation_type: CumulativeExplanationType::default(),
+                    propagation_method: CumulativePropagationMethod::TimeTablePerPoint,
```

Comment:
@maartenflippo - Remove this field from `CumulativeOptions` and create `CumulativePropagatorOptions`